js-activity/affordable-books.js

```diff
@@ -0,0 +1,14 @@
+// get all titles of books that is below $10 dollars.
+function getAffordableBooks(books){
+
+    
+   const affordable = books.map((book) =>{
+        if(book.price < 10){
+            return book.title
+        }
+    }).filter((book) => book!== undefined)
```

Comment:
```suggestion
   const affordable = books.filter((book) => book.price < 10).map(book => book.title);
```

this is much appropriate